apps/src/sites/studio/pages/schoolInfo.js

```diff
@@ -233,8 +242,42 @@ window.SchoolInfoManager = function (existingOptions) {
       clearAndHideDistrict();
       clearAndHideSchool();
       show('#school-name');
+    } else if (isUs() && isHomeschool()) {
```

Comment:
This is starting to get messy, but this is all going to get ripped apart as part of my current task (next PR), I'll try to simplify the logic then.